packages/strapi-plugin-content-manager/admin/src/components/SelectWrapper/index.js

```diff
@@ -145,8 +154,25 @@ function SelectWrapper({
     return inputValue;
   };
 
-  const onMenuScrollToBottom = () => {
-    setState(prevState => ({ ...prevState, _start: prevState._start + 20 }));
+  const handleMenuScrollToBottom = () => {
+    setState(prevState => ({ ...prevState, _limit: prevState._limit + 20 }));
+  };
+
+  const handleMenuClose = () => {
+    setState(prevState => ({ ...prevState, _contains: '' }));
```

Comment:
I think that we might need to reset the pagination to `0` since the idsToOmit might change I think that we should return to the initial state. What do you think?